distribution/src/main/resources/config.yml

```diff
@@ -60,12 +60,13 @@ mapdb-app-file-system:
 
 rao:
     default: SearchTreeRao
+    dc-mode: false
+    ac-to-dc-fallback: false
 
 linear-rao-parameters:
     max-number-of-iterations: 10
     security-analysis-without-rao: false
 
-search-tree-rao-paramters:
+search-tree-rao-parameters:
```

Comment:
@baptsegu @TheMaskedTurtle 
Do you think that this config.yml is reliable after this? :fearful:  
By the way, I think that even my modifications are wrong (because I put them in rao, and not rao-parameters)...

Reply:
Yeah, the new parameters should be in `search-tree-rao-parameters`.

Indeed, it means we never read this parameters when doing computation with our local config... we were using the default parameters (which are the same than the one below, that's why we didnt see the problem).

Thanks for the correction. If you want to make sure it works, you can change the local config and see in debug mode if the RaoParameters was changed accordingly.

Reply:
my bad, it should be in `rao-parameters`, not` search-tree-rao-parameters`